src/services/ledger/index.types.d.ts

```diff
@@ -57,6 +57,19 @@ type LnReceiveLedgerMetadata = NonIntraledgerLedgerMetadata & {
   hash: PaymentHash
 }
 
+type SwapFeeLedgerMetadata = {
+  swapAmount: Satoshis
+  swapId: string
+  htlcAddress: string
+  onchainMinerFee: Satoshis
+  offchainRoutingFee: Satoshis
+  serviceProviderFee: Satoshis
+  serviceProvider: string
+  currency: WalletCurrency
+  type: LedgerTransactionType
+  pending: boolean
```

Comment:
I think this will always be false?

```suggestion
  pending: false
```